pipeline/backend/docker/convert.go

```diff
@@ -69,8 +69,8 @@ func (e *docker) toConfig(step *types.Step, options BackendOptions) *container.C
 	return config
 }
 
-func toContainerName(step *types.Step) string {
-	return "wp_" + step.UUID
+func toContainerName(step *types.Step, workflowName string) string {
+	return "wp_" + workflowName + "-" + step.Name + "-" + step.UUID[:5]
```

Comment:
What kind of UUID are we using? Some have that first part is a timestamp component so that could be a problem, not sure if it is here tho

Reply:
It seems indeed that the first part is somehow static, at least on the last part changes on "restart" runs of the same pipeline.

Hence I will change it to the last 5 digits of the generated UUID.